src/schoology-extractor/schoology_extractor/helpers/sync.py

```diff
@@ -72,15 +72,18 @@ def _resource_has_changed(
     db_parsed_item = json.loads(json_response)
 
     for key in db_parsed_item:
-        api_value = resource[key]
-        db_value = db_parsed_item[key]
+        ignore_key = key == 'CreateDate' or key == 'LastModifiedDate'
```

Comment:
I often like to write this as `key in ["CreateDate", "LastModifiedDate"]`. What you have is perfectly fine too; I only mention it for situations with more than two values to check.